VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -104,6 +104,29 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         return '';
     }
 
+    getParticipantRowClasses(participant: any): string {
+        if (!this.isJohConsultation()) {
+            return 'participant-row';
+        }
+        if (!this.isLastJohFromGroupOrIndividualGroupMember(participant)) {
+            return 'participant-group-row';
+        } else {
+            return 'participant-row';
+        }
+    }
+
+    isLastJohFromGroupOrIndividualGroupMember(participant: any): boolean {
+        if (participant.hearing_role === HearingRole.PANEL_MEMBER) {
+            return this.panelMembers[this.panelMembers.length - 1].id === participant.id;
+        } else if (participant.hearing_role === HearingRole.WINGER) {
+            return this.wingers[this.wingers.length - 1].id === participant.id;
+        } else if (participant.hearing_role === HearingRole.JUDGE) {
+            return this.judge.id === participant.id;
```

Comment:
There will always only be one judge so this will always return true.

Reply:
true